### DISCUSSION TITLE
[FEATURE] Is there a way I can disable unused component

### DISCUSSION THREAD

**Discussion by sudipghimire533** [uPagge/uBlogger#111] [Q&A]
### Describe the feature you want

I absolutely love this theme and the thing I wish I could change is the way to remove unused component for good reason. I am yes new to hugo in general but as I see uBlogger theme is packed with features and In my case I dont use them all. Like In my case I got no reason to use Aplayer for my simple blog site and other components like mapbox are also just there for me I dont use them. So is there anything I can do about this?
-

**Comment by uPagge** [comment]
I will think about this in future versions
